packages/core/src/utils/editor.ts

```diff
@@ -65,7 +80,24 @@ export function checkHasEditorType(editor: EditorType): boolean {
   const commandConfig = editorCommands[editor];
   const commands =
     process.platform === 'win32' ? commandConfig.win32 : commandConfig.default;
-  return commands.some((cmd) => commandExists(cmd));
+  // First, try the standard commands
+  const hasStandardCommand = commands.some((cmd) => commandExists(cmd));
+  if (hasStandardCommand) {
+    return true;
+  }
+
+  // On macOS, try fallback path for vscode available
+  if (
+    editor === 'vscode' &&
+    process.platform === 'darwin' &&
```

Comment:
Lets not just check on darwin, lets check always, you can delete this bit and you should be able to delete the editor vscode check. SO fallback paths should be truly fall back paths and be checked always if they're configured

Reply:
Thanks for the suggestions, yes that makes sense actually, will do so.